Code/GraphMol/MolStandardize/Wrap/rdMolStandardize.cpp

```diff
@@ -72,6 +72,19 @@ void inPlaceHelper(RDKit::ROMol *mol, python::object params, FUNCTYPE func) {
   func(*static_cast<RDKit::RWMol *>(mol), *ps);
 }
 
+template <typename FUNCTYPE>
+void inPlaceHelper2(RDKit::ROMol *mol, python::object params,
+                    bool skip_standardize, FUNCTYPE func) {
+  if (!mol) {
+    throw_value_error("Molecule is None");
+  }
+  const RDKit::MolStandardize::CleanupParameters *ps =
+      &RDKit::MolStandardize::defaultCleanupParameters;
+  if (params) {
+    ps = python::extract<RDKit::MolStandardize::CleanupParameters *>(params);
+  }
+  func(*static_cast<RDKit::RWMol *>(mol), *ps, skip_standardize);
```

Comment:
So one can cast a `ROMol*` to `RWMol*` without Bad Consequences? I'll keep that in mind if I do not need a copy.

Reply:
Well, it is officially UB. But in the RDKit the only difference between RWMol and ROMol is that an RWMol has a few more methods. There are no additional data members, so I think that doing that cast is as safe as UB ever can be.